stories/Welcome.mdx

```diff
@@ -1,7 +1,158 @@
+          </div>
+        </div>
+        <div class="team-member flex flex-col gap-2">
+          <div>
+            <h4>Walter Furrer</h4>
+            <p>Documentation Engineer</p>
+          </div>
+          <div>
+            <a href="https://github.com/FurrerW" target="_blank" rel="noreferrer">GitHub</a>
+            <a href="https://www.linkedin.com/in/furrerw/" target="_blank" rel="noreferrer">LinkedIn</a>
+          </div>
+        </div>
+        <div class="team-member flex flex-col gap-2">
+          <div>
+            <h4>Danielle Lindblom</h4>
+            <p>Frontend Engineer</p>
+          </div>
+          <div>
+            <a href="https://github.com/Danielle254" target="_blank" rel="noreferrer">GitHub</a>
+            <a href="https://www.linkedin.com/in/Danielle-Lindblom" target="_blank" rel="noreferrer">LinkedIn</a>
+          </div>
+        </div>        
+        <div class="team-member flex flex-col gap-2">
+          <div>
+            <h4>Shashi Lo</h4>
+            <p>Engineering Manager</p>
+          </div>
+          <div>
+            <a href="https://github.com/shashilo" target="_blank" rel="noreferrer">GitHub</a>
+            <a href="https://www.linkedin.com/in/shashilo/" target="_blank" rel="noreferrer">LinkedIn</a>
+          </div>
+        </div>
+        <div class="team-member flex flex-col gap-2">
+          <div>
+            <h4>Corina Murg</h4>
+            <p>Frontend Engineer</p>
+          </div>
+          <div>
+            <a href="https://github.com/CorinaMurg" target="_blank" rel="noreferrer">GitHub</a>
+            <a href="https://www.linkedin.com/in/corinamurg/" target="_blank" rel="noreferrer">LinkedIn</a>
+          </div>
+        </div>
+        <div class="team-member flex flex-col gap-2">
+          <div>
+            <h4>Chris Nowicki</h4>
+            <p>Software Engineer</p>
+          </div>
+          <div>
+            <a href="https://github.com/chris-nowicki" target="_blank" rel="noreferrer">GitHub</a>
+            <a href="https://www.linkedin.com/in/chris-nowicki/" target="_blank" rel="noreferrer">LinkedIn</a>
+          </div>
+        </div>
+        <div class="team-member flex flex-col gap-2">
+          <div>
+            <h4>Mai Vang</h4>
+            <p>Frontend Engineer</p>
+          </div>
+          <div>
+            <a href="https://github.com/vmaineng" target="_blank" rel="noreferrer">GitHub</a>
+            <a href="https://www.linkedin.com/in/mai-vang-swe/" target="_blank" rel="noreferrer">LinkedIn</a>
+          </div>
+        </div>
+      </section>
+    </section>
+    <footer class="flex flex-row justify-between align-bottom debug">      
+      <a href="https://github.com/LetsGetTechnical/gridiron-survivor">
+        <Github class="w-10 h-auto duration-200 hover:text-zinc-50 hover:scale-125 hover:duration-200" /> 
+      </a>
+      <a href="./About/Application Setup.mdx" >
```

Comment:
This link is incorrect, please take a look at how the links in the other documentation are done.

Reply:
This should now be fixed.